amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/README.md

```diff
@@ -13,8 +15,16 @@ The conditions and case details are sumarized below:
 - Mesh size: 512 x 512 x 192
 - Total mesh size: 50331648 cells
 
+**Contents**
+
+- [Simulation setup](#simulation-setup)
+- [Performance](#performance)
+- [Results](#results)
+
 ## Simulation Setup
 
+Full details of the simulation setup are provided in [**setup documentation**](setup/README.md).
+
 The case was set up using the [AMR-Wind frontend](https://github.com/Exawind/amr-wind-frontend) with the notebook [convectiveABL_setup.ipynb](convective_abl_nrel5mw/setup/convectiveABL_setup.ipynb).  This allows the locations of the refinement regions and sampling planes to be setup properly relative to the location of the NREL5MW turbine.
```

Comment:
I'm getting a file not found for the notebook link.

Reply:
Fixed the link